tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/confirm_dialog.py

```diff
@@ -22,3 +22,7 @@ def tap_confirm(self, bookmark=False):
             *self._confirm_button_locator))
         Wait(self.marionette).until(expected.element_displayed(confirm))
         confirm.tap()
+        
+        from gaiatest.apps.homescreen.regions.bottom_bar import BottomBar
+        return BottomBar(self.marionette)
```

Comment:
The implicit rule I saw in the codebase is the following: 
Import globally, unless you're dealing with circular dependencies. For instance: Contacts calls the page region called EditContact, which can return Contacts again. In this case, you get the circular dependency.

Here, I agree, the bottom bar can be at the top